tests/integration/test_release_event.py

```diff
@@ -929,3 +930,95 @@ def test_dont_retry_propose_downstream_task(
     results = run_propose_downstream_handler(event_dict, package_config, job_config)
 
     assert not first_dict_value(results["job"])["success"]
+
+
+def test_dist_git_push_release_failed_issue_creation_disabled(
```

Comment:
JFC :eyes: such a simple thing /o we definitely need to factor out the reporting :D